axiomapy/axiomaapi/portfoliogroups.py

```diff
@@ -136,3 +136,23 @@ def delete_portfolio_group(
         _logger.info(f"Deleting at {url}")
         response = AxiomaSession.current._delete(url, return_response=return_response)
         return response
+
+    @staticmethod
+    def patch_portfolio_groups(portfolio_group_id: int,
+                               portfolios_dict: dict,
+                               return_response: bool = False):
+        """This method patches a portfolio group based on the json provided
+
+        Args:
+            portfolio_group_id: portfolio group to be deleted
+            portfolios_dict: portfolios to upsert or remove from the portfolio group
+            return_response: If set to true, the response will be returned
+
+        Returns:
+            Success message if the portfolio group is deleted successfully. Code 204
```

Comment:
Again, delete is not accurate.

Reply:
Hi Huang, please find the updates committed!